assets/src/edit-story/components/library/panes/text/textPane.js

```diff
@@ -64,14 +57,7 @@ function TextPane(props) {
         />
       )}
 
-      <Section
-        title={__('Presets', 'web-stories')}
-        titleTools={
```

Comment:
This is the only place where `titleTools` appears to have been used.

Let's remove support for this prop from the `Section` component too